bin/python/ripple/util/Sign.py

```diff
@@ -22,9 +23,9 @@
     check <key>
         Check an existing key for validity.
 
-    sign <sequence> <validator-public> <master-secret>
+    sign <sequence> <validator-public-key> <validator-private-key> <master-secret>
         Create a new signed manifest with the given sequence
-        number, validator public key, and master secret key.
+        number, validation public key, and master public key.
```

Comment:
I've updated `manifest-tool-guide.md` and cherry-picked @seelabs 's renaming commit. I further renamed `validator` to `validation` since `rippled validation_create` returns `validation_public_key` and `validation_private_key`